src/Ng/CommandHelpers.cs

```diff
@@ -284,24 +285,69 @@ public static StorageFactory CreateStorageFactory(IDictionary<string, string> ar
             else if (storageType.Equals("Azure", StringComparison.InvariantCultureIgnoreCase))
             {
                 string storageAccountName;
-                if (!arguments.TryGetValue("-storageAccountName", out storageAccountName))
+                string storageKeyValue;
+
+                bool useKeyVault = true;
+
+                string storageContainer;
+                if (!arguments.TryGetValue("-storageContainer", out storageContainer))
```

Comment:
Can we refactor this construct a bit? E.g. something like:

```
string storageContainer;
TryGetArgument(arguments, "-storageContainer", out storageContainer, required: true);
```

Makes the entire code block a bit more readable?